src/Dict.h

```diff
@@ -728,10 +729,9 @@ template <typename T> class Dictionary
 	/// The capacity of the table, Buckets + Overflow Size.
 	int Capacity(bool expected = false) const
```

Comment:
Given you're doing cleanups and I stumbled over that: Would you be up for removing the expected arg here? And either open-code the `bucket_count` access at the only caller that uses `expected=true` or introduce an `ExpectedCapacity()` helper? The callsite with `Capacity(true)` is hardly descriptive.